test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-test-helper
```

Comment:
Maybe this should go in a commit appart from the migration one

Reply:
@pedrobaeza Can you show us the way?